docs/SECURITY.md

```diff
@@ -1 +1,19 @@
 # CWA-Server Security
+
+## TLS Configuration
+
+The cwa-server uses encrypted connections for external and internal communication. The main configuration for the external communication channel can be found [here](https://github.com/corona-warn-app/cwa-server/blob/36ac6f7cbcf9b230d6af8ddea96b387b47a3dee9/services/submission/src/main/resources/application.yaml#L76). The current settings will only allow TLS version 1.2. and 1.3. The offered ciphers are chosen in accordance to the recommendations in [BSI-TR-02102-2](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR02102/BSI-TR-02102-2.pdf).
```

Comment:
I would suggest to avoid permalinks if possible since then there's less chance of the documentation needing update. In this instance for example, I would have a relative link to the `application.yaml` and ask the reader to refer to the `server` configuration. 

Why? Let's say that in future we add some configuration properties and now `server` configuration is on line 80 instead. The permalink would still point them to the `server` configuration on line 76 of an older version of the file which the reader may mistakenly take to be the latest file and miss on out on some new properties that may have been added. With a relative link, they would always see the file in the current repo but will have to do a quick search for 'server' themselves.